### PR TITLE
SC-6971 & SC-7582 - Fix grammar mistakes in nbc registration

### PR DESCRIPTION
**Pull request by CaptainUnbrauchbar**
# Description

Fixed two simple grammar mistakes in registration form and Registration Email text. Added a switch-case "instanceSpecificTitle" where future grammar exceptions can be added and used in "res.locals.theme.theme_title".
## Links to Tickets or other pull requests

https://ticketsystem.schul-cloud.org/browse/SC-6971
https://ticketsystem.schul-cloud.org/browse/SC-7582
## Changes

Fixes two grammar mistakes: "Niedersächsische Bildungscloud" -> "Niedersächsischen Bildungscloud".
## Data Security <sub><sup>details [on Confluence](https://docs.schul-cloud.org/x/2S3GBg)</sup></sub>

No Data Security Topics.
## Screenshots of UI changes

grafik
Normal Title displayed for default/boss Instance (HPI Schul-Cloud)
grafik
Modified Title displayed for n21/nbc Instance (Niedersächsischen Bildungscloud)
## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: helpers/theme.js [hunk 1]
```diff
@@ -6,14 +6,25 @@ const {
 	HOST,
 } = require('../config/global');
 
+const instanceSpecificTitle = () => {
```
Comment (CaptainUnbrauchbar) [inline helpers/theme.js:9]: Can be used to add future instance-specific grammar exceptions

File: helpers/theme.js [hunk 2]
```diff
@@ -6,14 +6,25 @@ const {
...
+	switch (SC_THEME) {
+		case 'n21':
+			return 'Niedersächsischen Bildungscloud';
```
Comment (CeEv) [inline helpers/theme.js:12]: What is the value of SC_TITLE for n21?

**Review by CeEv — CHANGES_REQUESTED** [review]

File: helpers/theme.js (hunk 2 above)
Comment (CaptainUnbrauchbar) [reply helpers/theme.js:12]: 'Niedersächsische Bildungscloud' from an old registration E-Mail. I implemented instanceSpecificTitle to allow exceptions in 
any instance if needed (just add another case)

Comment (CaptainUnbrauchbar) [reply helpers/theme.js:12]: Ive renamed the function to make its use more intuitive

**Comment by MartinSchuhmacher** [comment]
Closed due to externa contribution

**Pull request closed without merging**